core/src/templates/components/card/card.twig

```diff
@@ -3,78 +3,94 @@
  * @file
  * Card Component.
  *
- * Basic building block component with two variants
- *   - Horizontal: An image beside text content
+ * Basic building block component with the following variants:
  *   - Vertical (default): An image above text content
+ *   - Horizontal: An image beside text content
+ *   - Link: Whole card is a link with no button or call to action link
  *
  * Available variables:
  * - attributes: For additional HTML attributes not already provided.
- * - modifier_class: Additional css classes to change look and behaviour.
- * -- su-card--horizontal
+ * - modifier_class: Additional CSS classes to change look and behaviour.
  * - card_link: The href value for the card. A single card is only allowed to link to one URL.
  * - card_image: A full HTML entity for an image.
- * - card_super_headline: Text heading
+ * - card_super_headline: Text super headline. Often used for metadata such as category.
  * - card_headline: Text heading
  * - card_body: An open variable for anything. Generally this would be for
  *     WYSIWYG editor contents.
- * - card_cta_attributes: Any html attributes on the cta link that need to be
+ * - card_cta_attributes: Any html attributes on the link that need to be
  *     added. For example: _target.
  * - card_cta_label: The clickable text for the CTA link
  * - card_button_label: The text label of the button component.
  */
 #}
-<div{{ attributes }} class="su-card {{ modifier_class }}">
 
-  {# Card Image #}
-  {% if card_image is not empty -%}
-    {{ card_image }}
+{# For variant where whole card is a link, change the outer wrapper from <div> to <a> #}
+{% if "su-card--link" in modifier_class %}
+  {% set allow_links = false %}
+{% else %}
+  {% set allow_links = true %}
+{%- endif -%}
+
+<div {{ attributes }} class="su-card {{ modifier_class }}">
+  {%- if allow_links == false %}
+    <a href="{{ card_link }}" {{ card_cta_attributes }}>
   {%- endif -%}
 
-  <div class="su-card__contents">
+    {# Card Image #}
+    {% if card_image is not empty -%}
```

Comment:
Does the variable create extra whitespace? Seems a little redundant without any other markup around the variable.

Reply:
The if statement around the card_image? I think that was just there since the beginning. I'd be happy to remove it.

Reply:
🤔 Leave it alone for now. Thinking about this a little more, it may come in useful when we transition to a media space.